tools/convert.py

```diff
@@ -39,11 +40,12 @@ def load_state_dict(path):
 
     return sd
 
-def load_model(path):
+def load_model(path, force_arch=None):
     state_dict = load_state_dict(path)
-
     # from ComfyUI model detection
-    if "transformer_blocks.0.attn.norm_added_k.weight" in state_dict:
+    if force_arch is not None:
+        arch = force_arch
+    elif "transformer_blocks.0.attn.norm_added_k.weight" in state_dict:

```

Comment:
Add the proper key here for diffusers instead of the override. Something like `down_blocks.1.attentions.1.transformer_blocks.0.attn1.to_k.weight` should work.

Reply:
i wasn't sure whether you were referring to Flux or SD1 or SDXL here - the quoted section in the code is Flux but it would make more sense if you meant a non-Flux key. anyway, i didn't do anything with this yet.